plugins/toolkit/resources/resources/software/aws/toolkits/resources/MessagesBundle.properties

```diff
@@ -555,11 +555,11 @@ codemodernizer.chat.message.transform_cancelled_by_user=I cancelled your transfo
 codemodernizer.chat.message.transform_stopped_by_user=I stopped your transformation. If you want to start another transformation, choose **Start a new transformation**.
 codemodernizer.chat.message.transform_stopping=I'm stopping your transformation...
 codemodernizer.chat.message.validation.check_eligible_projects=I'm checking for open projects that are eligible for Code Transformation.
-codemodernizer.chat.message.validation.check_passed=I can upgrade your Java {0} module. To start the transformation, I need some information from you. Choose the module you want to upgrade and the target code version to upgrade to. Then, choose **Transform**.
+codemodernizer.chat.message.validation.check_passed=I can upgrade your Java {0} module. To start the transformation, I need some information from you. Choose the module you want to upgrade and the target code version to upgrade to. Then, choose **Transform**.\n\nIf you don't see the module you want to transform, you might need to configure your project so I can find it. Go to **File** and then **Project Structure** and make sure the correct JDK is selected in the SDK field.
```

Comment:
Don't think we should use a mix of `module` and `project` -- why not just pick one and stick with it? I'd say `project` since we're directing them to `"Project" Structure`

Reply:
I do see in Intellij's documentation that "multi-module project" is mentioned. So I think they are still different concepts. 
https://www.jetbrains.com/help/idea/maven-support.html